gli/tags.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 import torch
 
-import gli
+import glb
```

Comment:
glb has been changed to gli. Could you download the latest version of the main repo and reinstall the python (conda or pip) environment to test the code? You could still run this code locally probably because your local python environment still has the old glb package installed.

Reply:
have replaced.
For testing, need to run `python3 tags.py --dataset cora --task NodeClassification `.